### PR TITLE
Rewrite code samples README for Node/Express sign in users - REVISION

### PR DESCRIPTION
**Pull request by kengaderdus**
## Purpose

With the rebranding enabling Microsoft Entra in Azure Portal, the inaccuracies in our READMEs are increasing daily. There's also the problem of duplication of content (similar content in README and learn docs), and customer being unable to find the product docs if they first land on the code samples. We're rewriting this README, also tag it to make it easier for customers to know the use case it addresses, and to link it to the docs. Similar feedback was given during CIAM developer journey. 

## Does this introduce a breaking change?

```
[ ] Yes
[-] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[-] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
Use new REAME instructions to run the sample
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* Verify that the new REAME instructions are valid

### CODE REVIEW

File: 1-Authentication/5-sign-in-express/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 page_type: sample
-name: A Node.js & Express web app authenticating users against Azure AD for Customers with MSAL Node
-description: This sample demonstrates a Node.js & Express web app authenticating users against Azure Active Directory Customer Identity Access Management (Azure AD for Customers) with Microsoft Authentication Library for Node (MSAL Node)
+name: Sign in users in a sample Node.js & Express web app by using Microsoft Entra External ID for customers
```
Comment (cilwerner) [inline 1-Authentication/5-sign-in-express/README.md:3]: Isn't the `for customers` going away with the "soon to be confirmed" final names. Just adding this as a check more than anything else

File: 1-Authentication/5-sign-in-express/README.md [hunk 2]
```diff
@@ -37,163 +39,97 @@ extensions:
...
-
-![Scenario Image](./ReadmeFiles/topology.png)
+|          Instruction  |                Description                 |
+|-----------------------|--------------------------------------------|
+| **Use case**          | This code sample applies to **customer configuration uses case**![Yes button](./ReadmeFiles/yes.png "Title"). If you're looking for a workforce configuration use case, use [Tutorial: Enable a Node.js (Express) application to sign in users by using Microsoft Entra ID](https://github.com/Azure-Samples/ms-identity-node)      |
+| **Scenario**        | Sign in users. You acquire an ID token by using authorization code flow with PKCE. |
+|    **Add sign in to your app**        | Use the instructions in [Sign in users in a Node.js web app](https://learn.microsoft.com/entra/external-id/customers/tutorial-web-app-node-sign-in-prepare-tenant) to learn how to add sign in to your Node web app. |
+|**Product documentation** | Explore [Microsoft Entra ID for customers documentation](https://review.learn.microsoft.com/entra/external-id/customers/) |
 
 ## Contents
```
Comment (cilwerner) [inline 1-Authentication/5-sign-in-express/README.md:53]: Really like this.

File: 1-Authentication/5-sign-in-express/README.md [hunk 3]
```diff
@@ -37,163 +39,97 @@ extensions:
...
-1. The **ID Token** proves that the user has successfully authenticated against **Azure AD for Customers**.
-
-![Scenario Image](./ReadmeFiles/topology.png)
+|          Instruction  |                Description                 |
+|-----------------------|--------------------------------------------|
+| **Use case**          | This code sample applies to **customer configuration uses case**![Yes button](./ReadmeFiles/yes.png "Title"). If you're looking for a workforce configuration use case, use [Tutorial: Enable a Node.js (Express) application to sign in users by using Microsoft Entra ID](https://github.com/Azure-Samples/ms-identity-node)      |
+| **Scenario**        | Sign in users. You acquire an ID token by using authorization code flow with PKCE. |
+|    **Add sign in to your app**        | Use the instructions in [Sign in users in a Node.js web app](https://learn.microsoft.com/entra/external-id/customers/tutorial-web-app-node-sign-in-prepare-tenant) to learn how to add sign in to your Node web app. |
+|**Product documentation** | Explore [Microsoft Entra ID for customers documentation](https://review.learn.microsoft.com/entra/external-id/customers/) |
```
Comment (cilwerner) [inline 1-Authentication/5-sign-in-express/README.md:51]: Great!

**Review by cilwerner — COMMENTED** [review]
I really like this. It gets to the point, redirects to Learn documentation, and emphasises the code segments well.

**Comment by kengaderdus** [comment]
#assign-reviewer: @garrodonnell

**Comment by kengaderdus** [comment]
@kleggMS, would appreciate a review and your thoughts.

File: 1-Authentication/5-sign-in-express/README.md (hunk 3 above)
Comment (arbaazwahid) [inline 1-Authentication/5-sign-in-express/README.md:51]: is this link correct. can external users access it?(review.learn.microsoft)

File: 1-Authentication/5-sign-in-express/README.md [hunk 4]
```diff
@@ -37,163 +39,97 @@ extensions:
...
-### Step 2: Install project dependencies
-
-```console
-    cd 1-Authentication\5-sign-in-express\App
-    npm install
-```
+## Register the web application in your tenant
 
-### Step 3: Register the sample application(s) in your tenant
+You can register an app in your tenant automatically by using Microsoft Graph PowerShell or Manually register your apps in Microsoft Entra Admin center.
```
Comment (arbaazwahid) [inline 1-Authentication/5-sign-in-express/README.md:72]: is it possible to put the instructions in bold for powershell and for entra admin. i was trying the steps and got confused which are entra admin steps and which are powershell steps.

File: 1-Authentication/5-sign-in-express/README.md
Comment (arbaazwahid) [inline 1-Authentication/5-sign-in-express/README.md:1]: really good additions to the readme files. like the additions of separating the workforce tenant, instructions and scenarios.

File: 1-Authentication/5-sign-in-express/README.md (hunk 4 above)
Comment (kengaderdus) [reply 1-Authentication/5-sign-in-express/README.md]: Not possible, but I have changed formatting 

image

**Comment by kengaderdus** [comment]
Dear reviewers, thank you for your comments.

**Pull request merged**